pom.xml

```diff
@@ -198,7 +200,15 @@ THE SOFTWARE.
       <version>1.7</version>
       <type>jar</type>
       <optional>true</optional><!-- no need to have this at runtime -->
+    </dependency> 
+    <dependency>
+      <groupId>org.jenkins-ci</groupId>
+      <artifactId>annotation-indexer</artifactId>
+      <version>1.12</version>
```

Comment:
Does this match the version used by the Jenkins core version you are targeting?

Reply:
Yes, 2.73.2 uses this version